docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE public.cities (
+    name text,
+    population real,
+    elevation integer
+);
+
+CREATE TABLE public.capitals (
+    state character(2) NOT NULL
+)
+INHERITS (public.cities);
+```
+
+---
+
+### %Type syntax is not supported
+
+**GitHub**: [Issue #23619](https://github.com/yugabyte/yugabyte-db/issues/23619)
+
+**Description**: If you have any function, procedure, or trigger using the `%TYPE` syntax for referencing a type of a column from a table, then it errors out in YugabyteDB with the following error:
+
+```output
+ERROR: invalid type name "employees.salary%TYPE" (SQLSTATE 42601)
+```
+
+**Workaround**: Fix the syntax to include the actual type name instead of referencing the type of a column.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE public.employees (
+    employee_id integer NOT NULL,
+    employee_name text,
+    salary numeric
+);
+
+
+CREATE FUNCTION public.get_employee_salary(emp_id integer) RETURNS numeric
+    LANGUAGE plpgsql
+    AS $$
+DECLARE
+    emp_salary employees.salary%TYPE;  -- Declare a variable with the same type as employees.salary
+BEGIN
+    SELECT salary INTO emp_salary
+    FROM employees
+    WHERE employee_id = emp_id;
+
+    RETURN emp_salary;
+END;
+$$;
+```
+
+Suggested change to CREATE FUNCTION is as follows:
+
+```sql
+CREATE FUNCTION public.get_employee_salary(emp_id integer) RETURNS numeric
+    LANGUAGE plpgsql
+    AS $$
+DECLARE
+    Emp_salary NUMERIC;  -- Declare a variable with the same type as employees.salary
+BEGIN
+    SELECT salary INTO emp_salary
+    FROM employees
+    WHERE employee_id = emp_id;
+
+    RETURN emp_salary;
+END;
+$$;
+```
+
+---
+
+### GIN indexes on multiple columns are not supported
+
+**GitHub**: [Issue #724](https://github.com/yugabyte/yb-voyager/issues/724)
+
+**Description**: If there are GIN indexes in the source schema on multiple columns, as it is not supported by YugabyteDB, it results in an error during import schema as follows:
+
+```output
+ERROR: access method "ybgin" does not support multicolumn indexes (SQLSTATE 0A000)
+```
+
+**Workaround**: Currently, as there is no workaround, modify the schema to not include such indexes.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE public.test_gin_json (
+    id     integer,
+    text   jsonb,
+    text1  jsonb
+);
+
+CREATE INDEX gin_multi_on_json
+    ON public.test_gin_json USING gin (text, text1);
+```
+
+---
+
+### Policies on users in source require manual user creation
+
+**GitHub**: [Issue #1655](https://github.com/yugabyte/yb-voyager/issues/1655)
+
+**Description**: If there are policies in the source schema for users in database, the users have to be created manually on target YugabyteDB, as currently migration USER/GRANT is not supported. Skipping the manual user creation will return an error in import schema as follows:
```

Comment:
If there are policies in the source schema for users in the database, the USERs have to be created manually on target YugabyteDB, as currently the migration of USER/GRANT is not supported. Skipping the manual user creation will return an error in import schema as follows: